support/cas-server-support-cassandra-ticket-registry/src/test/java/org/apereo/cas/utils/TicketCreator.java

```diff
@@ -20,7 +20,7 @@
  *
  * @since 5.1.0
  */
-public class TicketCreator {
+public final class TicketCreator {
```

Comment:
@mmoayyed, as you can see in this change, I tried with and without the final modifier. What am I missing?